suitcase/tiff/tests.py

```diff
@@ -1,2 +1,67 @@
-# Tests should generate (and then clean up) any files they need for testing. No
-# binary files should be included in the repository.
+from . import export
+import numpy
+from numpy.testing import assert_array_equal
+import pytest
+from suitcase.utils.conftest import (simple_plan,
+                                     multi_stream_one_descriptor_plan,
+                                     one_stream_multi_descriptors_plan)
+import tempfile
+import tifffile
+
+@pytest.fixture()
+def expected():
+    expected_array = numpy.ones((10, 10))
+
+    return expected_array
+
+@pytest.fixture(params=[True, False], scope='function')
+def stack_images(request):
+    return request.param
+
+
+def test_simple_plan(events_data, expected, stack_images):
+    ''' runs a test using a simple count plan with num=5
+    '''
+    run_plan_test(simple_plan, events_data, expected, stack_images)
+
+
+def test_multi_stream_one_descriptor_plan(events_data, expected, stack_images):
+    ''' runs a test using a simple count plan with num=5
+    '''
+    run_plan_test(multi_stream_one_descriptor_plan, events_data, expected,
+                  stack_images)
+
+
+def test_one_stream_multi_descriptors_plan(events_data, expected, stack_images):
+    ''' runs a test using a simple count plan with num=5
+    '''
+    run_plan_test(one_stream_multi_descriptors_plan, events_data, expected,
+                  stack_images)
+
+
+def run_plan_test(plan, events_data, expected, stack_images):
+    ''' runs a test using the plan that is passed through to it
+
+    ..note::
+
+        Due to the `events_data` `pytest.fixture` this will run multiple tests
+        each with a range of detectors and a range of event_types. see
+        `suitcase.utils.conftest` for more info
+
+    '''
+
+    collector = events_data(simple_plan)
```

Comment:
should this be `event_data(plan)`?

Reply:
good catch, thanks @tacaswell I have updated this